models/staging/stripe/stg_payments.sql

```diff
@@ -3,7 +3,8 @@ SELECT
     ORDERID AS ORDER_ID,
     PAYMENTMETHOD AS PAYMENT_METHOD,
     STATUS,
-    AMOUNT,
+    AMOUNT AS RAW_AMOUNT,

```

Comment:
Since the rest of your project is in lowercase, I recommend updating this model as well.

Reply:
@amychen1776 
Committed new version with lower case